packages/server/src/synchronizer/rules/config/index.ts

```diff
@@ -0,0 +1,49 @@
+import File from 'vinyl'
+import {resolve} from 'path'
+import {Rule} from '../types'
+import {fileTransformStream} from '../pipeline'
+
+type Args = {
+  srcPath: string
+  entries: string[]
+}
+
+function createStubNextConfig(path: string) {
+  return new File({
+    path: resolve(path, 'next.config.js'),
+    contents: Buffer.from(`module.exports = {}`),
+  })
+}
+
+function createBlitzConfigLoader(path: string) {
+  const blitzConfigLoader = `
+const {withBlitz} = require('@blitzjs/server');
+const config = require('./blitz.config.js');
+module.exports = withBlitz(config);  
+`
```

Comment:
This doesn't look as good, but it won't add the first empty line causing prettier not to complain and reformat it

```suggestion
  const blitzConfigLoader = `const {withBlitz} = require('@blitzjs/server');
const config = require('./blitz.config.js');
module.exports = withBlitz(config);  
`
```